<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGRA - Lector de etiquetas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="button_agregar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Columnas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reducir">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reducir columnas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_imprimir">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_desagregar">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Desagregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda_Agra"/>
    <addaction name="actionAcerca_de_Agra_v1_0"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAyuda_Agra">
   <property name="text">
    <string>Ayuda Agra</string>
   </property>
  </action>
  <action name="actionAcerca_de_Agra_v1_0">
   <property name="text">
    <string>Acerca de Agra v1.0</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
